### PR TITLE
Task5

### PR DESCRIPTION
**Pull request by yana-may25**

### CODE REVIEW

File: task5.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+function range(start, end, step) {
+    if (step === undefined) {
+        step = 1;
+    }
+    let range = new Array(Math.round((Math.abs(start - end) + 1) / step)), sum = 0;
+    if (start > end) {
+        step = -step;
+    }
+    for (let i = 0; i < range.length; i++, start += step) {
```
Comment (MrOlegKravchenko) [inline task5.js:9]: Think of `i = start` to short code a little.

File: task5.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+function range(start, end, step) {
+    if (step === undefined) {
+        step = 1;
+    }
+    let range = new Array(Math.round((Math.abs(start - end) + 1) / step)), sum = 0;
```
Comment (MrOlegKravchenko) [inline task5.js:5]: Implementation is a bit complex for this function.

File: task5.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    if (start > end) {
```
Comment (MrOlegKravchenko) [inline task5.js:6]: Optional task: put this into the **for** condition

File: task5.js [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+    return range;
+}
+
+function sum(range) {
+    let sum = 0;
+    for (let i = 0; i < range.length; i++) {
+        sum += range[i];
+    };
+    return sum;
+}
```
Comment (MrOlegKravchenko) [inline task5.js:22]: Could you please left some console.logs so I can test your code right away ?

File: task5.js (hunk 1 above)
Comment (MrOlegKravchenko) [reply task5.js]: yes

Comment (yana-may25) [reply task5.js]: sure
stupid of me to ask

Comment (MrOlegKravchenko) [reply task5.js]: it's all okay

**Review by MrOlegKravchenko — APPROVED** [review]

**Pull request merged**